src/FTL.xml

```diff
@@ -0,0 +1,148 @@
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Redistribution</p>
+          <p>This license grants a worldwide, royalty-free, perpetual and irrevocable right and license to
+             use, execute, perform, compile, display, copy, create derivative works of, distribute and
+             sublicense the FreeType Project (in both source and object code forms) and derivative works
+             thereof for any purpose; and to authorize others to exercise some or all of the rights granted
+             herein, subject to the following conditions:</p>
+          <list>
+            <li>
+              <b>o</b>
+              <p>Redistribution of source code must retain this license file (`FTL.TXT') unaltered; any
+                 additions, deletions or changes to the original files must be clearly indicated in
+                 accompanying documentation. The copyright notices of the unaltered, original files must be
+                 preserved in all copies of source files.</p>
+            </li>
+            <li>
+              <b>o</b>
+              <p>Redistribution in binary form must provide a disclaimer that states that the software is based
+                 in part of the work of the FreeType Team, in the distribution documentation. We also encourage
+                 you to put an URL to the FreeType web page in your documentation, though this isn't
+                 mandatory.</p>
+            </li>
+          </list>
+          <p>These conditions apply to any software derived from or based on the FreeType Project, not just
+             the unmodified files. If you use our work, you must acknowledge us. However, no fee need be
+             paid to us.</p>
+        </li>
+        <li>
+          <b>3.</b>
+          <p>Advertising</p>
+          <p>Neither the FreeType authors and contributors nor you shall use the name of the other for
+             commercial, advertising, or promotional purposes without specific prior written
+             permission.</p>
+          <p>We suggest, but do not require, that you use one or more of the following phrases to refer to
+             this software in your documentation or advertising materials: `FreeType Project', `FreeType
+             Engine', `FreeType library', or `FreeType Distribution'.</p>
+          <p>As you have not signed this license, you are not required to accept it. However, as the FreeType
+             Project is copyrighted material, only this license, or another one contracted with the
+             authors, grants you the right to use, distribute, and modify it. Therefore, by using,
+             distributing, or modifying the FreeType Project, you indicate that you understand and accept
+             all the terms of this license.</p>
+        </li>
+        <li>
+          <b>4.</b>
+          <p>Contacts</p>
+        </li>
+      </list>
+      <p>There are two mailing lists related to FreeType:</p>
```

Comment:
no, sam had submitted a PR against kris' repo, but neither kris nor I had merged it. 

I merged it just now, so the current SPDX PR reflects the fixed version (with mailing list paragraphs moved up a bit and into the &lt;li&gt; for Contacts.